bootstrap.sh

```diff
@@ -141,12 +158,19 @@ install(){
     nix-channel --add https://github.com/LnL7/nix-darwin/archive/master.tar.gz darwin || exit
     nix-channel --update || exit
 
-    # Copy the example configuration
+    # Set up the configuration
     if [ ! -e "$HOME/.nixpkgs/darwin-configuration.nix" ]; then
-      echo -e "Copying example configuration to "$YELLOW"~/.nixpkgs/darwin-configuration.nix"$ESC"..."
-      mkdir -p "$HOME/.nixpkgs" || exit
-      cp "$HOME/.nix-defexpr/channels/darwin/modules/examples/simple.nix" "$HOME/.nixpkgs/darwin-configuration.nix" || exit
-      chmod u+w "$HOME/.nixpkgs/darwin-configuration.nix" || exit
+        if [ ! -z $CONFIGURATION ]; then
+            echo -e "Copying $CONFIGURATION to "$YELLOW"~/.nixpkgs/darwin-configuration.nix"$ESC"..."
+            curl $CONFIGURATION > "~/.nixpkgs/darwin-configuration.nix"
```

Comment:
It’d be nice if this also supported pointing at a directory, and copying all the contents to ~/.nixpkgs/.